### PR TITLE
protected accolades for authenticated and active users

### PR DESCRIPTION
**Pull request by 20was**

### CODE REVIEW

File: src/modules/permission/NoPermissionPage.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+const NoPermissionPage = () => {
+    return (
+        <div className="grid h-screen place-content-center bg-white px-4">
+            <div className="text-center">
+                <Image
+                    src={"/images/illustrations/notFound.svg"}
+                    width="1024"
+                    height="768"
+                    alt="Believe in power"
```
Comment (Aaasisss) [inline src/modules/permission/NoPermissionPage.tsx:12]: Small, Alt text change please

File: src/hooks/useSlackUserStatus.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+const useSlackUserStatus = (userEmail?: string) => {
+    return useQuery({
+        queryKey: ['slackUserStatus', userEmail],
+        queryFn: async () => {
+            const response = await axios.get(
+                `${process.env.NEXT_PUBLIC_SLACK_API_ROOT_PATH}/slack/user/status`,
+                {
+                    params: {userEmail},
+                    headers: {
+                        'x-api-key': process.env.NEXT_PUBLIC_SLACK_API_TOKEN,
```
Comment (Aaasisss) [inline src/hooks/useSlackUserStatus.ts:13]: Maybe we need to create api routes to hide the api keys

**Comment by Aaasisss** [comment]
Small change please

**Review by Aaasisss — COMMENTED** [review]
Just a small change please

**Comment by 20was** [comment]
also fixes #58

**Comment by 20was** [comment]
Fixes #59 too. @boltdynamics @sonam-serchan

File: src/components/layout/MobileNavigation/MobileNav.tsx [hunk 3]
```diff
@@ -1,44 +1,50 @@
...
+                    "translate-x-full": !isMobileMenuOpen,
+                    "translate-x-0": isMobileMenuOpen,
+                },
+            )}
+        >
+            <nav aria-label="Mobile Global" className="p-2 text-sm">
+                <ul className="flex h-screen w-full flex-col items-center justify-center gap-4 p-3">
+
+                    {menus.map(({name, href, isProtected}) => (
+                        (!isProtected || isActiveSlackUser) && (
```
Comment (boltdynamics) [inline src/components/layout/MobileNavigation/MobileNav.tsx:34]: Ah nice!

**Review by yashcrest — COMMENTED** [review]
LGTM

File: src/app/api/slack/userStatus/route.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+      {
+        params: { userEmail },
+        headers: {
+          "x-api-key": process.env.SLACK_API_TOKEN,
+        },
+      },
+    );
+
+    return NextResponse.json(response.data, { status: 200 });
+  } catch (error) {
```
Comment (Aaasisss) [inline src/app/api/slack/userStatus/route.ts:26]: maybe we need to log the errors so we know what went wrong.
maybe later

Comment (thorn-pod) [reply src/app/api/slack/userStatus/route.ts:26]: The next line is returning error but not printing in console.

**Pull request merged**